Utils.ps1

```diff
@@ -26,6 +27,98 @@ function Invoke-Utf8ConsoleCommand ([ScriptBlock]$cmd) {
     }
 }
 
+function Add-ImportModuleToProfile {
+    param (
+        [Parameter(Position = 0, Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $ProfilePath,
+
+        # This is only required to support PS v2 there $PSScriptRoot only works in a .psm1 file
+        [Parameter(Position = 1, Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $ScriptRoot
+    )
+
+    # Check if profile script exists
+    if (Test-Path -LiteralPath $profilePath) {
+        $profileContent = @(Get-Content -LiteralPath $profilePath)
+        $profileContent += [string]::Empty # Empty line between previous profile script and import statement
+        $encoding = Get-FileEncoding $profilePath
+    }
+    else {
+        # Doesn't exist, so create it
+        New-Item -Path $profilePath -ItemType File
+        $profileContent = @()
+        $encoding = 'ascii'
+    }
+
+    # Check if the location of this module file is in the PSModulePath
+    if (Test-InModulePath $ScriptRoot) {
+        $profileContent += "Import-Module posh-git"

```

Comment:
Doh, that makes the Pester test a little harder.  Oh well, hopefully not too much harder.  :-)